templates/image.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+
+export default class Image extends Component {
+	render () {
+
+		const createMarkup = () => {
+			return 	{__html: 	`<!--[if IE 9]><video style="display: none;"><![endif]-->
+								${srcset.xl ? '<source srcSet=' + url + srcset.xl + ' media="(min-width: 1220px)"/>' : null}
+								${srcset.l ? '<source srcSet=' + url + srcset.l + ' media="(min-width: 980px)"/>' : null}
+								${srcset.m ? '<source srcSet=' + url + srcset.m + ' media="(min-width: 740px)"/>' : null}
+								${srcset.s ? '<source srcSet=' + url + srcset.s + ' media="(min-width: 490px)"/>' : null}
+								<!--[if IE 9]></video><![endif]-->
+								${srcset.fallback
+									? '<img class="' + imgClass + 'n-image__img" src=' + url + srcset.fallback + ' alt="' + alt + '"/>'
+									: srcset.default
+										? '<img class="' + imgClass + 'n-image__img" srcSet=' + url + srcset.default + ' alt="' + alt + '"/>'
+										: '<img class="' + imgClass + 'n-image__img" alt="' + alt + '"/>'
+								}`
+					};
+		};
+
+		const picClass = this.props.picClass ? `${this.props.picClass} ` : "";
```

Comment:
```
`${this.props.picClass} `
```
```
this.props.picClass + ' '
```
are exactly the same length in characters.  string interpolation vs. concatenation are basically identical so i think either is fine…  are there another reason why not to use interpolation?